ui/src/components/Search.js

```diff
@@ -1,39 +1,117 @@
 import React from "react";
+import "@clr/icons";
+import "@clr/icons/shapes/essential-shapes.min.js";
+import "@clr/icons/clr-icons.css";
 import AsyncSelect from "react-select/lib/Async";
+import { components } from "react-select";
+
+import DownAngleIcon from "libs/icons";
 
 // To get readable class names, add classNamePrefix="foo" to <AsyncSelect>
-const customStyles = {
+const styles = {
+  clearIndicator: (provided, state) => ({
+    ...provided,
+    color: "#5c912e"
+  }),
   container: (provided, state) => ({
     ...provided,
     fontFamily: ["Montserrat", "sans-serif"].join(","),
     fontSize: 13
   }),
   control: (provided, state) => ({
     ...provided,
-    // Remove 1 pixel border to the left and right of search box
-    border: 0
+    backgroundColor: "#ebedef",
+    border: 0,
+    height: 45,
+    margin: "9px 15px 0px 15px"
+  }),
+  dropdownIcon: {
+    color: "#5c912e",
+    height: 30,
+    marginRight: 11,
+    width: 30
+  },
+  dropdownIndicator: (provided, state) => ({
+    ...provided,
+    padding: "8px 16px 8px 14px"
+  }),
+  indicatorsContainer: (provided, state) => ({
+    ...provided,
+    height: 45
+  }),
+  indicatorSeparator: (provided, state) => ({
+    ...provided,
+    height: 32
+  }),
+  input: (provided, state) => ({
+    ...provided,
+    marginLeft: 0
   }),
-  // Left-align search box text with dataset name
   option: (provided, state) => ({
     ...provided,
-    paddingLeft: 27
+    backgroundColor: state.isFocused ? "#f1f4f8" : "white",
+    borderBottom: "1px solid #e2e7ee",
+    color: "#333f52",
+    fontSize: 14,
+    fontWeight: 600,
+    lineHeight: "21px",
+    paddingLeft: 13
+  }),
+  menu: (provided, state) => ({
+    ...provided,
+    margin: "5px 15px 0 15px",
+    width: "-webkit-calc(100% - 30px)",
+    width: "-moz-calc(100% - 30px)",
+    width: "calc(100% - 30px)"
+  }),
+  menuList: (provided, state) => ({
+    ...provided,
+    // Don't show horizontal scrollbar
+    overflowX: "hidden",
+    paddingTop: 0
   }),
-  // Chip background color
+  // multiValue = Chip styling
   multiValue: (provided, state) => ({
     ...provided,
-    backgroundColor: "#ebedef"
+    backgroundColor: "#74ae43",
+    borderRadius: "15.5px",
+    height: "31",
+    margin: "0 15px 0 0"
+  }),
+  multiValueLabel: (provided, state) => ({
+    ...provided,
+    color: "white",
+    fontSize: 12,
+    fontWeight: 500,
+    padding: "8px 21px 0 20px",
+    // For some reason paddingLeft from above is ignored; put again here
```

Comment:
I wonder if you would experience this issue with other browsers?

Reply:
Not sure, and also not sure what you're getting at...?